billboard.java

```diff
@@ -0,0 +1,56 @@
+//package usaco2018january;
+//All 10 tests work no errors
+//50 min to complete
+//Difficulty may have been a 7/10
+//I thought I did okay but I didn't read the question cafefully and wasted 20 minutes overthinking the question

```

Comment:
That is a good reflection! So the lesson is to always spend time reading and understanding the problems. There is no second step if you didn't do your first step well.

There is this saying -- "Slow is faster". I would always sketch out the problem on paper and going through and solve the example with pencil before writing first line of code.